pulse/broker/src/main/java/gov/ca/emsa/pulse/service/SearchService.java

```diff
@@ -76,8 +76,8 @@ public SearchService() {
 		HashMap<String, String> customAttributes = new HashMap<String,String>();
 		customAttributes.put("RequesterFirstName", user.getFirstName());
 		customAttributes.put("RequestReason", "Patient is bleeding.");
-		customAttributes.put("PatientGivenName", toSearch.getGivenName());
-		customAttributes.put("PatientFamilyName", toSearch.getFamilyName());
+		customAttributes.put("PatientGivenName", toSearch.getPatientName().getGivenName().get(0).getGivenName());
+		customAttributes.put("PatientFamilyName", toSearch.getPatientName().getFamilyName());
```

Comment:
Should just pass the entire array of PatientNames to the service, and let the service pass that along to the external entities that are responding